json/src/lib.rs

```diff
@@ -219,6 +219,66 @@ pub struct GetBlockHeaderResult {
     pub next_block_hash: Option<bitcoin::BlockHash>,
 }
 
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub struct GetBlockStatsResult {
+    #[serde(rename = "avgfee")]
+    pub avg_fee: u32,
```

Comment:
IIUC there's a preference by the folks here to use `u64` over `u32`/`usize`. Maybe @stevenroose can explain more.

Reply:
I suppose this can also be an `Amount`, right?

Reply:
Oh right, this could. But I was referring to all the `u32` fields, not just that one

Reply:
Would appreciate some more info about 'using `u64` over `u32`/`usize` _for all fields_'. @stevenroose ?

Reply:
Just that we use 64-bit numbers for amounts. But in this crate, we are mostly using the dedicated `Amount` type. You can find some other usages of that type and the serde-tag you have to use with it.